package.json

```diff
@@ -24,10 +24,9 @@
    },
    "devDependencies": {
       "@commitlint/cli": "7.5.2",
```

Comment:
For posterity: ADIRL, we'll take this out since the travis-cli dependency has this as a dependency. That way upgrades are easier (the two are in sync).